Sources/Logging.php

```diff
@@ -501,6 +501,10 @@ function logActions($logs)
 			$smcFunc['db_free_result']($request);
 		}
 
+		$always_log = array('agreement_accepted', 'policy_accepted', 'agreement_updated', 'policy_updated');
+		if ((empty($modSettings['modlog_enabled']) && !in_array($action, $always_log)) || !isset($log_types[$log_type]))
```

Comment:
Notice: Undefined variable: log_type in C:\wamp64\www\SMF2.1\Sources\Logging.php on line 505